### PR TITLE
OSSMDOC-533: merge service mesh attributes files

### PR DESCRIPTION
**Pull request by JStickler**
xref:#44067 for distributed tracing attributes

[OSSMDOC-218](https://issues.redhat.com/browse/OSSMDOC-218) - Update the attributes to avoid confusion
[OSSMDOC-553](https://issues.redhat.com/browse/OSSMDOC-553) - Merge the two service mesh attributes files into the OpenShift common-attributes file.

Updates all attributes to unique names.
Moves attributes from independent files to common-attributes
Removes two independent attribute files.
Updates attribute references in all service mesh v1 and v2 assemblies.

Preview for v2 starts here https://deploy-preview-44070--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-about.html

Preview for v1 starts here https://deploy-preview-44070--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v1x/servicemesh-release-notes.html

### CODE REVIEW

**Comment by JStickler** [comment]
FYI @abrennan89  and @rh-max  here's the second part of OSSMDOC-533

**Comment by JStickler** [comment]
FYI @ousleyp, not sure if you used any of the Service Mesh attributes in the Virtualization docs?

**Review by kalexand-rh — COMMENTED** [review]
This looks great! I have one question, then this looks good to go. :) Thank you!

File: _attributes/common-attributes.adoc [hunk 1]
```diff
@@ -71,8 +71,27 @@ endif::[]
...
 :logging-title: logging subsystem for Red Hat OpenShift
 :logging-title-uc: Logging subsystem for Red Hat OpenShift
 :logging: logging subsystem
 :logging-uc: Logging subsystem
+//service mesh v2
+:product-dedicated: Red Hat OpenShift Dedicated
+:SMProductName: Red Hat OpenShift Service Mesh
+:SMProductShortName: Service Mesh
+:SMProductVersion: 2.1.1
+:MaistraVersion: 2.1
```
[the remark this answers is not in the document]
Comment (JStickler) [reply _attributes/common-attributes.adoc:94]: Ah, you're right.  I'll remove it.

**Pull request merged**